libraries/disp/viewers/multiview.cpp

```diff
@@ -120,6 +120,8 @@ MultiViewWindow* MultiView::addWidgetBottom(QWidget* pWidget,
     pDockWidget->setWindowTitle(sName);
     pDockWidget->setWidget(pWidget);
 
+    pWidget->setParent(pDockWidget);
```

Comment:
This is interesting. I did some reading. pDockWidget's setWidget function does not take ownership of pWidget. So setting the parent here makes sense. Does this alone fix the destructor calling problem? Or do we still need to set the Qt::WA_DeleteOnClose attribute?